supplementary_style_guide/glossary_terms_conventions/general_conventions/e.adoc

```diff
@@ -13,6 +13,16 @@
 // Fuse: Prefixed second sentence with "In Red Hat Fuse, an EIP is a" (Breda)
 // Fuse: Changed "Camel" to "Red Hat Fuse" (Breda)
 // Fuse: Removed the final sentence about the Palette (Breda)
+
+[[egress]]
+==== image:images/yes.png[yes] egress (adjective, noun)
```

Comment:
From looking at other entries, the "noun" and "adjective" defintions are separate. WOuld you be OK to follow the same pattern?